src/core/CCHeap.ml

```diff
@@ -403,3 +412,22 @@ module Make(E : PARTIAL_ORD) : S with type elt = E.t = struct
          pp_elt out x)
       h
 end
+
+module Make_from_compare(E : TOTAL_ORD) =
+struct
```

Comment:
this would be more concise as `module Make_from_compare(E:TOTAL_ORD) =
   Make(struct type t = E.t … end)`

no need for inclusion :-)

Reply:
This syntax is pretty cool